src/Metadata/GetCollection.php

```diff
@@ -94,10 +87,81 @@ public function __construct(
         $provider = null,
         $processor = null,
         array $extraProperties = [],
-        ?string $itemUriTemplate = null
+
+        private ?string $itemUriTemplate = null,
```

Comment:
```suggestion
        private ?string $itemUriTemplate = null,
```

can this be protected?

Reply:
This can be changed but it would have no effect since the class is `final`.

Reply:
mhh right, I forgot that our end attributes where final. Note to self: this needs an interface.